src/main/java/org/teamresistance/frc/Robot.java

```diff
@@ -1,13 +1,47 @@
 package org.teamresistance.frc;
 
 import edu.wpi.first.wpilibj.IterativeRobot;
+import org.strongback.Strongback;
 
 /**
  * Main robot class. Override methods from {@link IterativeRobot} to define behavior.
  */
 public class Robot extends IterativeRobot {
 
+    Drive drive;
+
     @Override
     public void robotInit() {
+//      System.out.println(BuildConfig.AGENT);
+
+      Strongback.configure().recordNoEvents().recordNoData().initialize();
+      drive = new Drive(IO.robotDrive);
     }
+
+  @Override
+  public void autonomousInit() {
+    Strongback.start();
+  }
+
+  @Override
+  public void autonomousPeriodic() {
+
+  }
+
+  @Override
+  public void teleopInit() {
+
```

Comment:
We need `Strongback.start()` again here, according to the [docs](https://github.com/strongback/strongback-java/blob/master/strongback/src/org/strongback/Strongback.java#L85). 👍